### PR TITLE
Remove explicit dep on jar-dependencies

### PR DESCRIPTION
**Pull request by donoghuc**
Jruby ships with a version of this gem. Managing with bundler causes load issues. 

See https://github.com/elastic/ingest-dev/issues/4691

Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

**Comment by donoghuc** [comment]
```
➜  generated git:(5717f64) ✗ openssl x509 -in server_from_root.crt -noout -dates
notBefore=Nov 24 10:21:30 2021 GMT
notAfter=Nov 24 10:21:30 2024 GMT
➜  generated git:(5717f64) ✗ pwd
/Users/cas/elastic-repos/logstash-input-http/spec/fixtures/certs/generated
```

File: spec/fixtures/certs/generate.sh [hunk 1]
```diff
@@ -41,5 +41,8 @@ openssl pkcs12 -export -in client_from_root.crt -inkey client_from_root.key -out
 # use java keytool to convert all pkcs12 keystores to jks-format keystores (pass:12345678)
 keytool -importkeystore -srckeystore client_from_root.p12 -srcstoretype pkcs12 -srcstorepass 12345678 -destkeystore client_from_root.jks -deststorepass 12345678 -alias client_from_root
 
+# create pkcs8 version of root key
+openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in root.key -out root.key.pkcs8
```
Comment (donoghuc) [inline spec/fixtures/certs/generate.sh:45]: I noticed running the `generate.sh` script deleted a file called `root.key.pkcs8`. I'm assuming that is related to
```
logstash-1  |   1) LogStash::Inputs::Http request handling with json codec with ssl should parse the json body
logstash-1  |      Got 0 failures and 2 other errors:
logstash-1  | 
logstash-1  |      1.1) Failure/Error: Unable to find sun.security.rsa.RSAKeyFactory.engineGeneratePrivate(sun/security/rsa/RSAKeyFactory.java to read failed line
logstash-1  |           
logstash-1  |           Java::JavaSecuritySpec::InvalidKeySpecException:
logstash-1  |             java.security.InvalidKeyException: Unable to decode key
logstash-1  |           # sun.security.rsa.RSAKeyFactory.engineGeneratePrivate(sun/security/rsa/RSAKeyFactory.java:253)
logstash-1  |           # java.security.KeyFactory.generatePrivate(java/security/KeyFactory.java:389)
logstash-1  |           # jdk.internal.reflect.DirectMethodHandleAccessor.invoke(jdk/internal/reflect/DirectMethodHandleAccessor.java:103)
logstash-1  |           # java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:580)
logstash-1  |           # usr.share.plugins.plugin.spec.inputs.http_spec.<main>(/usr/share/plugins/plugin/spec/inputs/http_spec.rb:20)
logstash-1  |           # usr.share.plugins.plugin.spec.inputs.http_spec.setup_server_client(/usr/share/plugins/plugin/spec/inputs/http_spec.rb:497)
logstash-1  |           # usr.share.plugins.plugin.spec.inputs.http_spec.<main>(/usr/share/plugins/plugin/spec/inputs/http_spec.rb:43)
logstash-1  |           # java.lang.invoke.MethodHandle.invokeWithArguments(java/lang/invoke/MethodHandle.java:733)
logstash-1  |           # ------------------
logstash-1  |           # --- Caused by: ---
logstash-1  |           # Java::JavaIo::EOFException:
logstash-1  |           #   not enough content
logstash-1  |           #   java.base/sun.security.util.DerValue.<init>(DerValue.java:429)
logstash-1  | 
logstash-1  |      1.2) Failure/Error: Unable to find java.base/sun.security.rsa.RSAKeyFactory.engineGeneratePrivate(RSAKeyFactory.java to read failed line
logstash-1  |           
logstash-1  |           Java::JavaSecuritySpec::InvalidKeySpecException:
logstash-1  |             java.security.InvalidKeyException: Unable to decode key
logstash-1  |           # java.base/sun.security.rsa.RSAKeyFactory.engineGeneratePrivate(RSAKeyFactory.java:253)
logstash-1  |           # java.base/java.security.KeyFactory.generatePrivate(KeyFactory.java:389)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:106)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yield(Block.java:189)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.each(RubyArray.java:1981)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # usr.share.plugins.plugin.spec.inputs.http_spec.RUBY$block$\=\^main\_$0(/usr/share/plugins/plugin/spec/inputs/http_spec.rb:20)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.callDirect(CompiledIRBlockBody.java:141)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.MixedModeIRBlockBody.callDirect(MixedModeIRBlockBody.java:103)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.IRBlockBody.call(IRBlockBody.java:64)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.call(Block.java:148)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc.call(RubyProc.java:375)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.ProcMethod.call(ProcMethod.java:66)
logstash-1  |           # org.jruby.dist/org.jruby.ir.runtime.IRRuntimeHelpers.unresolvedSuper(IRRuntimeHelpers.java:1488)
logstash-1  |           # org.jruby.dist/org.jruby.ir.runtime.IRRuntimeHelpers.unresolvedSuperSplatArgs(IRRuntimeHelpers.java:1465)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.UnresolvedSuperInvokeSite.invoke(UnresolvedSuperInvokeSite.java:23)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:106)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yield(Block.java:189)
logstash-1  |           # org.jruby.dist/org.jruby.RubyHash.fetch(RubyHash.java:1441)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:106)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yield(Block.java:189)
logstash-1  |           # org.jruby.dist/org.jruby.RubyHash.fetch(RubyHash.java:1441)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.callDirect(CompiledIRBlockBody.java:141)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.IRBlockBody.call(IRBlockBody.java:64)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.call(Block.java:148)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc.call(RubyProc.java:375)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.ProcMethod.call(ProcMethod.java:66)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.failf(InvokeSite.java:802)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.failf(InvokeSite.java:816)
logstash-1  |           # usr.share.plugins.plugin.spec.inputs.http_spec.RUBY$block$\=\^main\_$0(/usr/share/plugins/plugin/spec/inputs/http_spec.rb:35)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.MixedModeIRBlockBody.yieldDirect(MixedModeIRBlockBody.java:111)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject.yieldUnder(RubyBasicObject.java:1736)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject.instance_exec(RubyBasicObject.java:2620)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject$INVOKER$i$0$0$instance_exec.call(RubyBasicObject$INVOKER$i$0$0$instance_exec.gen)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.fail(InvokeSite.java:777)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:106)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yield(Block.java:189)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.each(RubyArray.java:1981)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:106)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yield(Block.java:189)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.each(RubyArray.java:1981)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray$INVOKER$i$0$0$each.call(RubyArray$INVOKER$i$0$0$each.gen)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.JavaMethod$JavaMethodZeroBlock.call(JavaMethod.java:561)
logstash-1  |           # org.jruby.dist/org.jruby.RubyClass.finvokeWithRefinements(RubyClass.java:540)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject.send(RubyBasicObject.java:1713)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject$INVOKER$i$send.call(RubyBasicObject$INVOKER$i$send.gen)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:725)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.callDirect(CompiledIRBlockBody.java:141)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.IRBlockBody.call(IRBlockBody.java:64)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.call(Block.java:148)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc.call(RubyProc.java:317)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc$INVOKER$i$call.call(RubyProc$INVOKER$i$call.gen)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.MixedModeIRBlockBody.yieldDirect(MixedModeIRBlockBody.java:111)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject.yieldUnder(RubyBasicObject.java:1736)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject.instance_exec(RubyBasicObject.java:2620)
logstash-1  |           # org.jruby.dist/org.jruby.RubyBasicObject$INVOKER$i$0$0$instance_exec.call(RubyBasicObject$INVOKER$i$0$0$instance_exec.gen)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.fail(InvokeSite.java:777)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.callDirect(CompiledIRBlockBody.java:141)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.IRBlockBody.call(IRBlockBody.java:64)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.call(Block.java:148)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc.call(RubyProc.java:317)
logstash-1  |           # org.jruby.dist/org.jruby.RubyProc$INVOKER$i$call.call(RubyProc$INVOKER$i$call.gen)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.specialized.RubyArrayOneObject.collectArray(RubyArrayOneObject.java:267)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.map(RubyArray.java:2803)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:165)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:185)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.failf(InvokeSite.java:866)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:162)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:185)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.fail(InvokeSite.java:841)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.specialized.RubyArrayOneObject.collectArray(RubyArrayOneObject.java:267)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.map(RubyArray.java:2803)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:162)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:185)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.fail(InvokeSite.java:841)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.collectArray(RubyArray.java:2770)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.map(RubyArray.java:2803)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.collectArray(RubyArray.java:2770)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.map(RubyArray.java:2803)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.CompiledIRBlockBody.yieldDirect(CompiledIRBlockBody.java:151)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.BlockBody.yield(BlockBody.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.runtime.Block.yieldNonArray(Block.java:228)
logstash-1  |           # org.jruby.dist/org.jruby.specialized.RubyArrayTwoObject.collectArray(RubyArrayTwoObject.java:360)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray.map(RubyArray.java:2803)
logstash-1  |           # org.jruby.dist/org.jruby.RubyArray$INVOKER$i$0$0$map.call(RubyArray$INVOKER$i$0$0$map.gen)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.JavaMethod$JavaMethodZeroBlock.call(JavaMethod.java:561)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:725)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.YieldSite.yieldSpecific(YieldSite.java:157)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:725)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.YieldSite.yield(YieldSite.java:113)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:725)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:680)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:139)
logstash-1  |           # org.jruby.dist/org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:112)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.performIndirectCall(InvokeSite.java:735)
logstash-1  |           # org.jruby.dist/org.jruby.ir.targets.indy.InvokeSite.invoke(InvokeSite.java:657)
logstash-1  |           # java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
logstash-1  |           # org.jruby.dist/org.jruby.ir.Compiler$1.load(Compiler.java:114)
logstash-1  |           # org.jruby.dist/org.jruby.Ruby.runScript(Ruby.java:1245)
logstash-1  |           # org.jruby.dist/org.jruby.Ruby.runNormally(Ruby.java:1157)
logstash-1  |           # org.jruby.dist/org.jruby.Ruby.runFromMain(Ruby.java:983)
logstash-1  |           # org.jruby.dist/org.jruby.Main.doRunFromMain(Main.java:398)
logstash-1  |           # org.jruby.dist/org.jruby.Main.internalRun(Main.java:282)
logstash-1  |           # org.jruby.dist/org.jruby.Main.run(Main.java:227)
logstash-1  |           # org.jruby.dist/org.jruby.Main.main(Main.java:197)
logstash-1  |           # ------------------
logstash-1  |           # --- Caused by: ---
logstash-1  |           # Java::JavaIo::EOFException:
logstash-1  |           #   not enough content
logstash-1  |           #   java.base/sun.security.util.DerValue.<init>(DerValue.java:429)
```
Which I believe is the root cause of all the other failures. Trying to reverse engineer how that key is generated and how it is used...

**Comment by donoghuc** [comment]
Should be green except for the unrelated 7.current snapshot versions not existing 💚

File: spec/fixtures/certs/generate.sh (hunk 1 above)
Comment (robbavey) [reply spec/fixtures/certs/generate.sh:45]: Nice catch!

**Review by robbavey — APPROVED** [review]
LGTM

**Pull request merged**